src/python/nxt_python_asgi_lifespan.c

```diff
@@ -50,6 +52,11 @@ static PyMethodDef nxt_py_asgi_lifespan_methods[] = {
     { NULL, NULL, 0, 0 }
 };
 
+static PyMemberDef nxt_py_asgi_lifespan_members[] = {
+        {"state", T_OBJECT_EX, offsetof(nxt_py_asgi_lifespan_t, state), READONLY, PyDoc_STR("lifespan.state")},
+        {NULL, 0, 0, 0, NULL}
+};
```

Comment:
Not overly keen on how that looks.

Maybe it would be better to specify the struct members and lay it out to match the structure. By which I mean something like [this](https://elixir.bootlin.com/linux/latest/source/drivers/dma/sun4i-dma.c#L1295).

Reply:
but ```nxt_py_asgi_lifespan_methods``` use the same way, only change here makes it looks wired

Reply:
The two below are designated initialised, though they aren't arrays... OK, go with your version, but could you format it as

```c
static PyMemberDef nxt_py_asgi_lifespan_members[] = {                           
    { "state", T_OBJECT_EX, offsetof(nxt_py_asgi_lifespan_t, state), READONLY,
      PyDoc_STR("lifespan.state") },                                            
    { NULL, 0, 0, 0, NULL }                                                     
};
```

(spaces around {}, 4 char indent, PyDoc_STR() aligmnent...)

Thanks.